cmd/list.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import 	(
+	"fmt"
+	"os"
+	"github.com/spf13/cobra"
+	"github.com/sector-f/jh_extract/nvc"
+)
+
+var listCmd = &cobra.Command{
+	Use:   "list",
+	Short: "Manipulate nvc files",
+	Long: `based off jh_extract.py`,
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		reader, openErr := os.Open(args[0])
+		if openErr != nil {
+			fmt.Fprintln(os.Stderr, openErr)
+			os.Exit(1)
+		}
+		entries, listErr := nvc.ListEntries(reader)
+		if listErr != nil {
+			fmt.Fprintln(os.Stderr, listErr)
+			os.Exit(1)
+		}
+		for _, entry := range entries {
+			fmt.Printf("%v offset=%v %vB (%vB on disk) flags=%v\n",
+				nvc.Hash2String(entry.Hash),
+				entry.Offset,
+				entry.RawLength,
+				entry.Length,
+				entry.Flags)
```

Comment:
Might as well make `nvc.TocEntry` implement the `Stringer` interface at this point?

Reply:
:boom: